addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -346,13 +346,14 @@ PortalChatter.include({
         var messageIndex = $source.data("mes_index");
```

Comment:
Commit message:

Let's add a reference to the commit that introduced the issue, I believe it's https://github.com/odoo/odoo/commit/6303a3eacdca012649a2ffda627b65c17a7217f1